html2docx/html2docx.py

```diff
@@ -176,7 +180,20 @@ def handle_data(self, data: str) -> None:
     def handle_endtag(self, tag: str) -> None:
         if tag in ["a", "b", "em", "i", "span", "strong", "sub", "sup", "u"]:
             self.finish_run()
-        elif tag in ["h1", "h2", "h3", "h4", "h5", "h6", "li", "ol", "p", "pre", "ul"]:
+        elif tag in [
+            "blockquote",
+            "h1",
+            "h2",
+            "h3",
+            "h4",
+            "h5",
+            "h6",
+            "li",
+            "ol",
+            "p",
+            "pre",
+            "ul",
```

Comment:
This list is getting long. Perhaps it should move to a class variable. Something like `BLOCK_ELEMENTS`.